Gemfile

```diff
@@ -7,4 +7,16 @@ group :development, :test do
   gem 'rubocop-performance', require: false
   gem 'rubocop-rails', require: false
   gem 'rubocop-rspec', require: false
+
+  gem 'flowcommerce'
+  gem 'flowcommerce-reference'
+  gem 'rspec-rails', '~> 3.5'
+  gem 'spree_backend', '~> 2.3.0'
+
+  gem 'activerecord-postgres-json', '>= 0.2.3'
+  gem 'byebug'
+  gem 'countries'
+  gem 'factory_bot_rails', '~> 4.0'
+  gem 'faker'
```

Comment:
We don't need `faker`, because spree_backend requires spree_core, which has `ffaker`

Reply:
Thanks Aurel! didn't realize about that! I have just removed it!